template-parts/title.php

```diff
@@ -0,0 +1,7 @@
+<?php if( get_field('page_title') ): ?>
```

Comment:
Did you create a field for the user to enable or disable the title? So for example you would do the check
```
<?php if( get_field('enable_page_title') ): ?>
<?php if( get_field('page_title') ): ?>
...
<?php endif; ?>
<?php endif; ?>